services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceITest.kt

```diff
@@ -2719,9 +2722,18 @@ class HakemusServiceITest(
                         alluId,
                         ApplicationHistoryFactory.createEvent(
                             applicationIdentifier = identifier,
-                            newStatus = ApplicationStatus.DECISION)),
+                            newStatus = applicationStatus,
+                        ),
+                    ),
                 )
-            every { alluClient.getDecisionPdf(alluId) } returns PDF_BYTES
+            every {
```

Comment:
You can use the existing `mockAlluDownload` and `verifyAllyDownload`.

Reply:
Thank you for pointing this out. I missed these.